hedera-node/hedera-util-service-impl/src/main/java/com/hedera/node/app/service/util/impl/config/PrngConfig.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.util.impl.config;
+
+/**
+ * Configuration for the PRNG to check if it is enabled or not.
+ * @param isPrngEnabled true if the PRNG is enabled, false otherwise
+ */
+public record PrngConfig(boolean isPrngEnabled) {}
```

Comment:
I think it is kind of unusual to name a property `isSomething`. Verbs are usually reserved for methods. Is this something we also do elsewhere in the services-repository?

Reply:
I named it that way based on `utilPrng.isEnabled`. WIll rename to `prngEnabled`